gh_pages/_source/demo1/Inspect-the-package.md

```diff
@@ -10,6 +10,7 @@ cp -r ~/industrial_training/exercises/Perception-Driven_Manipulation/template_ws
 ```
 cd ~/perception_driven_ws/src/
 wstool update
+rosdep install --from-paths . --ignore-src -r -y
```

Comment:
re: `-r`, see https://github.com/swri-robotics/gilbreth/pull/13#discussion_r145325469.

Reply:
@gavanderhoorn this has been addressed